website/docs/d/partition.html.markdown

```diff
@@ -37,3 +38,4 @@ There are no arguments available for this data source.
 ## Attributes Reference
 
 `partition` is set to the identifier of the current partition.
+`url_suffix` is set to the URL Suffix associated with the current partition.
```

Comment:
Can you please add Markdown list syntax for these attributes and clarify what this attribute contains? 👍 

```suggestion
* `dns_suffix` is set to the base DNS domain name for the current partition (e.g. `amazonaws.com` in AWS Commercial, `amazonaws.com.cn` in AWS China).
```

Reply:
done